src/components/subscriptions/expiration/SubscriptionExpirationBanner.jsx

```diff
@@ -1,35 +1,69 @@
 
 import {
   SUBSCRIPTION_DAYS_REMAINING_MODERATE,
   SUBSCRIPTION_DAYS_REMAINING_SEVERE,
   SUBSCRIPTION_DAYS_REMAINING_EXCEPTIONAL,
 } from '../data/constants';
 import { SubscriptionDetailContext } from '../SubscriptionDetailContextProvider';
+import { configuration } from '../../../config';
+import { formatTimestamp } from '../../../utils';
 
-const SubscriptionExpirationBanner = () => {
-  const { subscription: { daysUntilExpiration }, hasMultipleSubscriptions } = useContext(SubscriptionDetailContext);
+const SubscriptionExpirationBanner = (props) => {
+  const {
+    subscription: { daysUntilExpiration, expirationDate },
+    hasMultipleSubscriptions,
+  } = useContext(SubscriptionDetailContext);
   const [showBanner, setShowBanner] = useState(true);
 
-  const renderMessage = () => (
-    <>
-      {hasMultipleSubscriptions && daysUntilExpiration <= 0 ? (
+  const renderMessage = () => {
+    // use subscription detail view messaging
+    if (props.subscriptionDetails) {
+      return (
         <>
-          This subscription cohort has expired. You may still view the statuses of learners who participated.
+          {hasMultipleSubscriptions && daysUntilExpiration <= 0 ? (
+            <>
+              <Alert.Heading>
+                This subscription plan&apos;s end date has passed
+              </Alert.Heading>
+              Administrative actions are no longer available as of the plan end date of May 31, 2021.
+              You may still view the statuses of your invited learners.
+            </>
+          ) : (
+            <>
+              <Alert.Heading>
+                This subscription plan&apos;s end date is approaching
+              </Alert.Heading>
+              Administrative actions will no longer be available fallowing the
+              plan end date on {formatTimestamp({ timestamp: expirationDate })}.
+            </>
+          )}
         </>
-      ) : (
-        <>
-          Your subscription is {daysUntilExpiration} days from expiration.
-          Contact the edX Customer Success team at
-          {' '}
-          <MailtoLink to="customersuccess@edx.org">customersuccess@edx.org</MailtoLink>
-          {' '}
-          to extend your contract.
-        </>
-      )}
-    </>
-  );
+      );
+    }
+    return (
+      <>
+        {hasMultipleSubscriptions && daysUntilExpiration <= 0 ? (
+          <>
+            <Alert.Heading>
+              Your subscription contract has expired
+            </Alert.Heading>
+            Renew your subscription today to reconnect your learning community.
+          </>
+        ) : (
+          <>
+            <Alert.Heading>
+              Your subscription contract is expiring soon
+            </Alert.Heading>
+            Your current subscription contract will expire in {daysUntilExpiration} days.
+            Renew your subscription today to minimize access disruption for your learners.
+          </>
+        )}
+      </>
+    );
+  };
```

Comment:
If I am reading this correctly the messaging really changes depending on if `props.subscriptionDetails` exists, but the blocks about `hasMultipleSubscriptions && daysUntilExpiration <= 0 ?` and when `hasMultipleSubscriptions` is false remain both when `props.subscriptionDetails` exists. I would suggest moving `props.subscriptionDetails` down further in to the conditional checks instead of having it on the outside. It should be easier to read and think about at that level.